touchforms/backend/touchcare.py

```diff
@@ -212,9 +212,9 @@ def filter_cases(filter_expr, api_auth, session_data=None, form_context=None,
                  restore_xml=None, force_sync=True, uses_sqlite=False):
     session_data = session_data or {}
     form_context = form_context or {}
-    modified_xpath = "join(',', instance('casedb')/casedb/case%(filters)s/@case_id)" % \
-                     {"filters": filter_expr}
 
+    modified_xpath = "join(',', instance('casedb')/casedb/case%(filters)s[@status= 'open']/@case_id)" % \
+                     {"filters": filter_expr}
```

Comment:
it's the same thing we do in the app manager: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/suite_xml/sections/entries.py#L65